chainer/links/connection/highway.py

```diff
@@ -0,0 +1,79 @@
+from chainer.functions.activation import relu
+from chainer.functions.activation import sigmoid
+from chainer import link
+from chainer.links.connection import linear
+
+
+class Highway(link.Chain):
+
+    """Highway module.
+
+    This is a network allowing unimpeded information flow across layer on
+    information highways.
+    It applies two different gates. One gate is transform gate
+    (:math:`tanh(W_h x + b_h)`), and the other is carry gate
+    (:math:`sigma(W_t x + b_t)`). Carry gate enable us to propagete large
+    gradient value to previous layer.
```

Comment:
One gate is ordinal non-linear transformation (:math:`activate(W_h x + b_h)`),
and the other is carry gate (:math:`1 - sigma(W_t x + b_t)`)